src/WooCommerce.php

```diff
@@ -1497,7 +1497,10 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
    *
    * @implements woocommerce_checkout_create_order
    */
-  public static function addCustomMetaForUser($order) {
+  public static function addCustomMetaForUser(WC_Order $order):void {
```

Comment:
```suggestion
  public static function addCustomMetaForUser(\WC_Order $order):void {
```

Reply:
I forgot the slash. Please don't forget to run the code again to see if everything works.

Reply:
Updated in the last commit.